.yarn/patches/chokidar-npm-3.5.3-c5f9b0a56a.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/index.js b/index.js
+index ed4b6d5b6cbabee76c879c6eedcc7bf6ba46d38d..8752893ca4986cf05103a46c438223b7ee4e1907 100644
+--- a/index.js
++++ b/index.js
+@@ -441,7 +441,7 @@ add(paths_, _origAdd, _internal) {
+ 
+   if (this.options.useFsEvents && this._fsEventsHandler) {
+     if (!this._readyCount) this._readyCount = paths.length;
+-    if (this.options.persistent) this._readyCount *= 2;
++    if (this.options.persistent) this._readyCount += paths.length;
```

Comment:
It seems like an upstream bug to me, because
`watcher.add(["a.js"]); watcher.add(["b.js"])` should yield the same `_readyCount` as calling `watcher.add(["a.js", "b.js"])`. However I am not familiar with the chokidar codebase at all, so I create an ad-hoc patch here and see if it is sufficient to save that test.

Reply:
The upstream PR: https://github.com/paulmillr/chokidar/pull/1288